client/editSchedule.js

```diff
@@ -1,9 +1,11 @@
 var editingRoundReact = new ReactiveVar(null);
 
 Template.editSchedule.helpers({
+  competition: function() {
+    return Competitions.findOne(this.competitionId);
+  },
   unscheduledRounds: function() {
-    var rounds = getUnscheduledRounds(this.competitionId);
-
+    var rounds = getScheduledRounds(this.competitionId, false);
```

Comment:
Calling getScheduledRounds() to get unscheduled rounds is not great naming.

There are a few ways to make calls like this readable. Maybe the most suitable here is

function getUnscheduledRounds(competitionId) { return _getRounds(competitionId, false); }
function getScheduledRounds(competitionId) { return _getRounds(competitionId, true); }

Other things I've dabbled with are.

 // this works great in Java with enums. Kinda weird here.
getRounds(this.competitionId, 'unscheduled');

// Takes some getting used to...
getRounds(this.competitionId, '{ scheduled: false }); 

// Not elegant, but simple:
getRounds(this.competitionId, false);